WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -163,6 +227,7 @@ struct AddEditCoupon: View {
             .navigationBarTitleDisplayMode(.large)
             .wooNavigationBarStyle()
         }
+        .ignoresSafeArea(.container, edges: [.horizontal, .bottom])
```

Comment:
Since we're not updating any padding with safe area insets, should we remove this?